src/main/app/src/utils/hakukohde/getFormValuesByHakukohde.ts

```diff
@@ -118,24 +116,22 @@ const getFormValuesByHakukohde = hakukohde => {
           toimitustapa,
           toimitusosoite,
           kuvaus = {},
-        }) => {
-          return {
-            tyyppi: { value: tyyppiKoodiUri },
-            nimi,
-            kuvaus,
-            toimitusaika: toimitusaika || '',
-            toimitustapa: {
-              tapa: toimitustapa || '',
-              paikka: {
-                osoite: _.get(toimitusosoite, 'osoite.osoite') || {},
-                postinumero: _.get(toimitusosoite, 'osoite.postinumeroKoodiUri')
-                  ? { value: toimitusosoite.osoite.postinumeroKoodiUri }
-                  : undefined,
-                sahkoposti: _.get(toimitusosoite, 'sahkoposti') || '',
-              },
+        }) => ({
+          tyyppi: { value: tyyppiKoodiUri },
+          nimi,
+          kuvaus: _.mapValues(kuvaus, parseEditorState),
+          toimitusaika: toimitusaika || '',
+          toimitustapa: {
+            tapa: toimitustapa || '',
+            paikka: {
+              osoite: _.get(toimitusosoite, 'osoite.osoite') || {},
```

Comment:
Oliko tarkoituksena muuten luopua pikkuhiljaa lodasin getteja ja käyttää optional chainingia?
```suggestion
              osoite: toimitusosoite?.osoite?.osoite || {},
```

Reply:
Joo, fiksailen